mithril-aggregator/src/certifier_service.rs

```diff
@@ -116,6 +124,9 @@ pub trait CertifierService: Sync + Send {
 
     /// Returns the list of the latest created certificates.
     async fn get_latest_certificates(&self, last_n: usize) -> StdResult<Vec<Certificate>>;
+
+    /// Verify the certificate chain starting with the given certificate.
+    async fn verify_certificate_chain(&self, epoch: Epoch) -> StdResult<()>;
```

Comment:
It looks like the description of the function does not match its signature :confused: